src/org/usfirst/frc/team2335/robot/subsystems/Climb.java

```diff
@@ -0,0 +1,38 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+
+public class Climb extends Subsystem 
+{
+    //Motor for climbing
+	Victor climbMotor;
+	
+	public Climb()
+	{
+		super();
```

Comment:
Ubspy is correct- super() calls the superclass's constructor method. 

As for what to do about this snippet, I'd run a quick test to see if removing it breaks anything. If nothing is broken as a result, feel free to remove it.